EnableJupyterDisplay.swift

```diff
@@ -0,0 +1,178 @@
+#if canImport(Cryptor)
+import Cryptor
+#endif
+
+import Foundation
+
+enum JupyterDisplay {
+    struct Header: Codable {
+        let msgId: String
```

Comment:
- `xxxId` -> `xxxID`.
- Avoid abbreviations.

```suggestion
        let messageID: String
```